app/common/patient/components/wizard/patientWizard.js

```diff
@@ -82,7 +82,7 @@
 
     function linkCancel() {
       if (updating) {
-        $state.go($stateParams.returnState, {patientUuid: vm.patient.uuid});
+        $state.go($stateParams.returnState, { patientUuid: vm.patient.uuid });
```

Comment:
Looks like you formatter is different from the rest of the the project. I'm looking for a solution for this. In the mean time please try [this](https://github.com/Microsoft/vscode/issues/18043).

Reply:
Yassin, as discussed, we should discuss this with the team. Perhaps when Guimino is back since he would also be interested in being part of this decision.